auth0/v3/management/rest.py

```diff
@@ -57,7 +55,6 @@ def post(self, url, data=None):
         headers = self.base_headers.copy()
         headers.update({
             'Authorization': 'Bearer %s' % self.jwt,
```

Comment:
it looks like the `authorization` header is present on all the http methods. I think this one should be part of base headers too